ServerCore/Pages/Events/Create.cshtml.cs

```diff
@@ -70,6 +72,12 @@ public async Task<IActionResult> OnPostAsync()
 
             await _context.SaveChangesAsync();
 
+            // Create base files for event content and style
+            await FileManager.UploadBlobAsync("global-styles.css", Event.ID, NewFileHelper.GetStreamContent(NewFileHelper.ContentType.GlobalCss), SharedResourceDirectoryName);

```

Comment:
Streams should be disposed after use

Reply:
MemoryStream documentation mentions in the Remarks section that it doesn't need explicit Dispose calls:  https://learn.microsoft.com/en-us/dotnet/api/system.io.memorystream?view=net-8.0